src/vs/workbench/contrib/positronPlots/browser/positronPlotsActions.ts

```diff
@@ -84,24 +96,113 @@ export class PlotsCopyAction extends Action2 {
 		});
 	}
 
+	private getItems(editorService: IEditorService): IQuickPickItem[] {
+		const items: IQuickPickItem[] = [
+			{
+				type: 'item',
+				id: CopyPlotTarget.VIEW,
+				label: localize('positronPlots.copyPlotsView', 'Plots View'),
```

Comment:
```suggestion
				label: localize('positronPlots.copyPlotsView', 'Active plots pane'),
```
nit: I'm not sure if we have a unified language around what we are calling each element in the sidebar, but "Plots View" felt a little vague

Reply:
Agreed we should standardize this. BTW I think VSCode tends to use title case for quickpick items.

Reply:
I checked this and everywhere else seems to use "<Name> View". So whether it's the *Session*, *Variables*, or *Console*, all the panes are referred to as a view.

![image](https://github.com/user-attachments/assets/64e91454-8e08-435b-b9f4-f090d294cc22)

I could change this to "From Plots View"? I think "Selected" is used to refer the visible plot in the view instead of "Active". The idea is to copy the currently selected plot from the view though.

Reply:
I think changing it to "From Plots View" would clarify which plot would be copied. Also, I'm changing the editor tab title back to the plot id. I agree that it's too confusing with using the code.